internal/clients/telegram/response/response.go

```diff
@@ -37,18 +168,155 @@ func (m SendMessage) SetParseMode(mode option.Option[string]) SendMessage {
 	return m
 }
 
+// EnableWebPreview enables the preview that is visible below the message and displays the webpage content.
+func (m SendMessage) EnableWebPreview() SendMessage {
```

Comment:
This was brought up in #6. If possible the commit that has this change should be moved to that branch